### ISSUE TITLE
tts:color not yet supported

### ISSUE THREAD

**Issue by palemieux** [skynav/ttt#154]
The following TTML:

```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<tt xml:lang="en" xmlns="http://www.w3.org/ns/ttml" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" ttp:timeBase="media" ttp:frameRate="24" ttp:frameRateMultiplier="1000 1001">
    <head>
        <styling>
            <style xml:id="basic" tts:color="white" tts:backgroundColor="transparent" tts:fontSize="100%" tts:fontWeight="normal" tts:fontStyle="normal"/>
        </styling>
        <layout>
            <region xml:id="bottom" tts:backgroundColor="transparent" tts:showBackground="whenActive" tts:origin="10% 10%" tts:extent="80% 80%" tts:displayAlign="after"/>
        </layout>
    </head>
    <body>
        <div>
            <p begin="00:00:00:00" end="00:00:02:00" region="bottom" style="basic" tts:textAlign="center">hello</p>
         </div>
    </body>
</tt>
```

generates the following SVG:

```
<?xml version="1.0" encoding="utf-8"?><svg height="720" width="1280" xmlns="http://www.w3.org/2000/svg"><rect fill="#000000" height="720" width="1280"/><g transform="translate(128,72)"><svg class="region" height="576" id="r1" width="1024"><g fill="#FFFF00" font-family="Noto Sans" font-size="1" transform="translate(0,574.75)"><g transform="translate(510.8486,1)"><g><text>h</text><text x="0.6182">e</text><text x="1.1821">l</text><text x="1.4399">l</text><text x="1.6978">o</text></g></g></g></svg></g></svg>
```

Note:

- fill="#FFFF00" (transparent instead of white characters)
- font-size="1" (shouldn't 100% translate to one cell height?)

**Comment by skynavga** [comment]
the presentation semantics of many TTML1 features are not yet supported in TTPE, including tts:color and the default initial value for font size of 1c; however, these features are scheduled for release 3

**Comment by skynavga** [comment]
Addressed in 956f249aeb1767a67dfda010e9d56bf766b3ac9b, 062626fd838058a1e14a1d560f55cdcd3ffb7c24, and a57180843b87f342cbe0a9b14d2fa7095e898032.
